.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,10 +7,13 @@ assignees: markdroth
 ---
 
 <!--
+PLEASE DO NOT POST A QUESTION HERE.
 This form is for bug reports and feature requests ONLY!
-For general questions and troubleshooting, please ask/look for answers here:
-- grpc.io mailing list: https://groups.google.com/forum/#!forum/grpc-io
+
+For general questions and troubleshooting, please ask/look for answers at
```

Comment:
The formatting here doesn't look like it will be like what you expect.  I suggest just splitting this into two paragraphs and omitting the bullets.